examples/multipart-upload-wrapper-example/README.md

```diff
@@ -0,0 +1,7 @@
+# An example web app shows how to use Here Data SDK for TypeScript to upload and publish large data in a browser.
+
+This is an HTML+JS UI and HTTP server with mocked routes to simulate the real DataStore.
+
```

Comment:
Please use the same structure as for the other examples. Add two sections: **Setup** and **Run the app**